src/js/components/FormField/FormField.js

```diff
@@ -261,13 +261,36 @@ const FormField = forwardRef(
           ) {
             wantContentPad = true;
           }
-          if (
+
+          const isInputComponent =
             child &&
             child.type &&
-            grommetInputNames.indexOf(child.type.displayName) !== -1 &&
+            grommetInputNames.indexOf(child.type.displayName) !== -1;
+          // inputs that would have their own focus indicator
+          // if theme.formField.focusIndicator is false
+          const wantInputFocusIndicator =
```

Comment:
Instead could we have something like this at the top of the file similar to how we set up `grommetInputNames` and `grommetInputPadNames`
```
const grommetInputFocusNames = [
  'CheckBox',
  'CheckBoxGroup',
  'RadioButton',
  'RadioButtonGroup',
  'RangeInput',
  'RangeSelector',
  'StarRating',
  'ThumbsRating',
];
```